src/Azure/Orleans.Streaming.EventHubs/Properties/AssemblyInfo.cs

```diff
@@ -1,7 +1,3 @@
 ﻿using System.Runtime.CompilerServices;
-using Orleans.CodeGeneration;
-using Orleans.ServiceBus.Providers;
 
 [assembly: InternalsVisibleTo("ServiceBus.Tests")]
-
-[assembly: KnownAssembly(typeof(EventHubSequenceTokenV2), TreatTypesAsSerializable = true)]
```

Comment:
How is a serializer generated for EventHubSequenceTokenV2?  It's not used in any grain interfaces.  That's why it was previously marked with the GenerateSerializer attribute

Reply:
It's marked as `[Serializable]`